pages/faq.mdx

```diff
@@ -11,7 +11,7 @@ We at [Determinate Systems][detsys] recommend using Nix **2.18.1** or newer in c
 You can upgrade Nix using the [`upgrade-nix`][upgrade-nix] command. To upgrade to Nix 2.18.1:
 
 ```shell
-sudo nix upgrade-nix \
+sudo -E env "PATH=$PATH" nix upgrade-nix \
```

Comment:
This doesn't look right. `-E` preserves the _entire_ environment, so there is no need to add `env ...` to the command.

Besides, it would be safer to preserve the just the `PATH` variable instead of the entire environment, as a good practice:
```suggestion
sudo --preserve-env=PATH nix upgrade-nix \
```